packages/amplify-category-auth/index.js

```diff
@@ -228,6 +229,10 @@ async function initEnv(context) {
   });
 
   const tasks = toBeCreated.concat(toBeUpdated);
+  // check if this initialization is happening on a pull
+  if (isPulling && allResources.length > 0) {
```

Comment:
Wouldn't all resource already contain resources to be created and deleted and to be updated?

Reply:
It won't be in created or updated resource list when doing a pull.
We also want to make sure we are adding it when a pull happens.